<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>purchasedate</class>
 <widget class="QMainWindow" name="purchasedate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Name">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameLine">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ISBN">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ISBN:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ISBNBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Purchase">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Purchased On:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="purchasedOnBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>381</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Category">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Category:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>381</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select Category</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OOP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Database Systems</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Artificial Intelligence</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="subLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SubCategory:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="subcomboBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>381</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select SubCategory</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ERD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SQL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OLAP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Data Mining</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C++</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Java</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Machine Learning</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Robotics</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Computer Vision</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="AuthorGBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>461</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Authors</string>
    </property>
    <widget class="QLabel" name="AuthorLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Author Name:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="authorsList">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>271</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="AuthorLine">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="TypeGBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>530</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Type</string>
    </property>
    <widget class="QRadioButton" name="RefBookButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Reference Book</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="TextBookButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Text Book</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="JournalButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Journal</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="IssuanceGBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>530</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Issuance</string>
    </property>
    <widget class="QCheckBox" name="Issued">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Issued</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Issued By:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IssuedBy">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Issued On:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="IssuedOn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="OkayButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>740</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Okay</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
